src/content/docs/en/reference/components-reference.mdx

```diff
@@ -0,0 +1,264 @@
+---
+title: Built-in Components Reference
+i18nReady: true
+---
+import Since from '~/components/Since.astro';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import ReadMore from '~/components/ReadMore.astro';
+
+## Built-in Components
+
+Astro includes several built-in components for you to use in your projects. All built-in components are available in `.astro` files via `import {} from 'astro:components';`.
+
+You can reference the `Props` of these components using the [`ComponentProps` type](/en/guides/typescript/#componentprops-type) utility.
+
+### `<Code />`
+
+```astro 'theme="dark-plus"' /wrap\b/ /(inline) 'defaultColor={false}' \/>/
```

Comment:
Sorry it seems I misplaced `defaultColor` in my previous suggestion... So both `inline` and `defaultColor` highlighting are broken. 😓 
````suggestion
```astro 'theme="dark-plus"' /wrap\b/ /(inline) \/>/ 'defaultColor={false}'
````
I think it should be good now... I don't know why I put it in the regex! 👀 (I should have done the review after and not before going to bed 😅 )